test/test_rest_vol.c

```diff
@@ -8254,23 +8253,24 @@ test_read_dataset_large_point_selection(void)
         goto error;
     }
 
-    for (i = 0, data_size = 1; i < DATASET_LARGE_READ_TEST_POINT_SELECTION_DSET_SPACE_RANK; i++)
-        data_size *= dims[i];
-    data_size = DATASET_LARGE_READ_TEST_POINT_SELECTION_DSET_DTYPESIZE;
```

Comment:
Looks like all that was needed to fix this was to change this to `data_size *= DATASET_LARGE_READ_TEST_POINT_SELECTION_DSET_DTYPESIZE;`. That said, I've asked @glennsong09 to remove the large I/O tests as we move these tests back into the library since they aren't exactly practical to run.